packages/ui/src/components/file-toolbar-actions.tsx

```diff
@@ -17,7 +17,7 @@ export const FileToolbarActions: FC<FileToolbarActionsProps> = ({
   showEdit = false
 }) => {
   return (
-    <ButtonGroup spacing="0" className="h-full rounded">
+    <ButtonLayout className="gap-0">
```

Comment:
This kinda looks like a better fit for the other `ButtonGroup` we had spoken about. For now, I don't think it benefits from `ButtonLayout`; I'd consider swapping for a `Layout.Horizontal`